documentation/data-protection.md

```diff
@@ -6,8 +6,19 @@ The [Data Protection Act](https://www.gov.uk/data-protection/the-data-protection
 
 ## What does data protection have to do with open data?
 
+In general, [open data](https://theodi.org/what-is-open-data) should not contain personal or sensitive personal data that could allow a living person to be identified. Data published to the [360Giving standard](http://www.threesixtygiving.org/standard/) should be anonymised to protect privacy as outlined in the ICO's [anonymisation code of practice](https://ico.org.uk/for-organisations/guide-to-data-protection/anonymisation/). There are cases where publishing personal data is in the public interest or where data can be published with the consent of the individual. We explore those cases below.
 
-[Open data](https://theodi.org/what-is-open-data) should not contain personal or sensitive personal data that could allow a living person to be identified (there are [exceptions](https://www.gov.uk/government/publications/dft-senior-officials-business-expenses-and-meetings-april-to-june-2014) but these don't apply to [360Giving](http://www.threesixtygiving.org/)). Data published to the [360Giving standard](http://www.threesixtygiving.org/standard/) should be anonymised to protect privacy as outlined in the ICO's [anonymisation code of practice](https://ico.org.uk/for-organisations/guide-to-data-protection/anonymisation/).
+## Publishing personal data with 360Giving
+We encourage publishers to carefully consider the value of sharing any personal data as part of their 360Giving publication and discourage sharing of sensitive personal data. 
+
+Before making a decision on including personal data:
+
+ 1. Review the ICO's [Key definitions of the data protection act](https://ico.org.uk/for-organisations/guide-to-data-protection/key-definitions/) to understand the difference between non-personal data, personal data and sensitive personal data;
+ 2. Review the ICO's [Guide to data protection](https://ico.org.uk/for-organisations/guide-to-data-protection/) to understand your organisation's obligations;
+ 3. Ensure your organisation has the [power to share the data](https://ico.org.uk/for-organisations/guide-to-data-protection/data-sharing/); 
+ 4. Restrict personal data to names used in official capacity, for example the contact names at the funding and recipient organisations, names of recipients of funds e.g. scholarships and identifiers that are open or in the public domain, for example [orcid](https://orcid.org/);
```

Comment:
I'd suggest breaking this into another section:

"Once you have decided to publish personal data:
1. Restrict..."